app/src/main/java/fi/sabriina/urbanhuikka/MainActivity.kt

```diff
@@ -233,13 +237,22 @@ class MainActivity : AppCompatActivity(), OnCardSwipeListener {
     }
 
     private fun cardSkipped() {
-        splashScreenManager.showSplashScreen(currentPlayer.name, currentPlayerPicture,"Ota ${currentCard!!.points} huikkaa!", drawableDrink)
-        endTurn()
+        CoroutineScope(Dispatchers.Main).launch {
+            val assistingPlayer = gameStateViewModel.getAssistingPlayer()
+            val assistingPlayerPicture = ContextCompat.getDrawable(this@MainActivity, currentPlayer.pictureResId)!!
+            splashScreenManager.showSplashScreen(currentPlayer.name, currentPlayerPicture,"Ota ${currentCard!!.points} huikkaa!", drawableDrink)
+            splashScreenManager.showSplashScreen(assistingPlayer.name, assistingPlayerPicture,"Ota ${currentCard!!.points} huikkaa!", drawableDrink)
```

Comment:
Assisting player gets points for every card even though they don't participate in the current card.